### ISSUE TITLE
Use Pulse API to fetch data

### ISSUE THREAD

**Issue by mmmavis** [mozilla/network-pulse#308]
Currently we get data directly from Google Spreadsheet. When we have successfully migrated data to our real database we should use Pulse API to fetch data instead.

Note: This is currently blocked by https://github.com/mozilla/network-pulse/issues/295 (Migrate data to real backend )

**Comment by mmmavis** [comment]
https://pulse-react.herokuapp.com/latest is now pulling data from Pulse API instead of Google Spreadsheet 🍾 :clinking_glasses: Once we have migrated spreadsheet data over we should be able to see a ton of entries on the site 😃 

Note that the current codebase is a bit messy as a few API endpoints are not available yet (e.g., endpoint to return just the featured entries etc) so I added some client side filtering code to serve as a interim solution. I'd like to keep this ticket open to remind me to tidy up the codebase once the endpoints are there.

**Comment by arlissc** [comment]
@mmmavis - curious - what data are you pulling from spreadsheets now ? can I take a peak ?
thanks :)

**Comment by mmmavis** [comment]
@arlissc hey 😄 for sure, here's the link: https://docs.google.com/spreadsheets/d/1vmYQjQ9f6CR8Hs5JH3GGJ6F9fqWfLSW0S4dz-t2KTF4/edit . We used it as the temporary and quick "database" solution. Since now our real database for Pulse is built we are gonna move away from Google Spreadsheet.

**Comment by mmmavis** [comment]
After a few PRs on Pulse API repo get merged I'll remove the code that does client side data filtering and switch to use the newly added endpoints/query params instead.

- https://github.com/mozilla/network-pulse-api/pull/32
- https://github.com/mozilla/network-pulse-api/pull/33
- ( https://github.com/mozilla/network-pulse-api/pull/35 )

**Comment by mmmavis** [comment]
Um I think creating a new ticket for the followup todos makes more sense. Let me close this ticket now as now Pulse does pull data from the API and not Google Spreadsheet. 🍾

**Comment by arlissc** [comment]
@mmmavis - awesome quick response. 👍
